.github/ISSUE_TEMPLATE/story.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Story
+about: Issue describing development work to fulfill a feature request
+title: ''
+labels: story, needs-triage
+assignees: ''
+priority: ''
+---
+
+### Description
+_What's the goal of this unit of work? What is included? What isn't included?_
+
+### Acceptance Criteria
+_What tasks need to be accomplished to achieve the goal?_
+
+### Design Consideration/Limitations
+_Why is this the route we should take to achieve our goal?_
+_What can't be achieved within this story?_
+
+### Dependencies
+_Do any other teams or parts of the New Relic product need to be considered?_
+_Some common areas: UI, collector, documentation_
+
+### Additional context
+_What else should we know about this story that might not fit into the other categories?_
+
+### Estimates
```

Comment:
do we really want to put that in the ticket?

Reply:
removed the estimate section. we can revisit later if needed.